src/Connection/RelayMethods.php

```diff
@@ -32,7 +32,7 @@ public function onFlushed(?callable $callback)
      * @param  string   $pattern
      * @return bool
      */
-    public function onInvalidated(?callable $callback, ?string $pattern = null)
+    public function onInvalidated(?callable $callback, string $pattern = null)
```

Comment:
Can you explain this change, or link me to the relevant docs article?

Reply:
It's related to php-cs-fixer v3.17 release, looks like right now rule set `@Symfony` (the one that we're using) includes new rule, so it doesn't require question mark on type-hint if `null` value is pre-defined.

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6876